resources/public/api/conf/3.0/common/headers.yaml

```diff
@@ -63,12 +63,32 @@ components:
         example: "-"
       required: false
 
-    deprecationHeader:
-      name: "Deprecation"
+    linkHeader:
+      name: "Link"
       in: header
       description: |
-        Specifies the deprecation status of an endpoint.
+        See the API documentation:
       schema:
         type: string
-        example: "This endpoint is deprecated. See the API documentation: https://developer.service.hmrc.gov.uk/api-documentation/docs/api/service/self-assessment-bsas-api"
+        example: "https://developer.service.hmrc.gov.uk/api-documentation/docs/api/service/self-assessment-bsas-api"
       required: false
+
+      deprecationHeader:
+        name: "Deprecation"
+        in: header
+        description: |
+          The deprecation date/time for this endpoint.
+        schema:
+          type: string
+          example: "Sun, 01 Jan 2023 23:59:59 GMT"
+        required: false
+
+      sunsetHeader:
+        name: "Sunset"
+        in: header
+        description: |
+          The earliest date/time this endpoint will become unavailable after deprecation.
+        schema:
+          type: string
+          example: "Sun, 02 Jul 2023 23:59:59 GMT"
+        required: false
```

Comment:
These lines need to be indented one tab less (so that they align with linkHeader). Otherwise the descriptions don't appear in the rendered docs. 

Likewise for the v4 headers.yaml